app/components/SearchBox/SearchBox.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { View, TextInput } from 'react-native';
+import { IconSearch } from '../icons/IconSearch';
+
+export const SearchBox: React.FC = (): JSX.Element => {
+    return (
+        <View
+            style={{
```

Comment:
use StyleSheet. it has optimizations. don't use inline styling if you mustn't do it.
It's the same as 
`<div style={{ height: 55px; }}>  ` - you don't do such thing in React.